internal/console/helpers/helpers.go

```diff
@@ -176,3 +178,26 @@ func CustomConsoleWriter(fileLogger *zerolog.ConsoleWriter) zerolog.ConsoleWrite
 
 	return *fileLogger
 }
+
+// FileAnalyzer determines the type of extension in the passed config file by its content
+func FileAnalyzer(path string) string {
+	ostat, _ := os.Open(path)
+	rc, _ := ioutil.ReadAll(ostat)
+	var temp map[string]interface{}
+
+	if err := json.Unmarshal(rc, &temp); err == nil {
+		return "json"
+	}
+
+	if err := yaml.Unmarshal(rc, &temp); err == nil {
+		return "yaml"
+	}
+
+	if c, err := hcl.Parse(string(rc)); err == nil {
+		if err = hcl.DecodeObject(&temp, c); err == nil {
+			return "hcl"
+		}
+	}
+
+	return "toml"
```

Comment:
What if I write something that is neither of the above and is not toml as well?
Maybe in that case we should report an error?